### DISCUSSION TITLE
Button layer clarification "super and undo will be unavailable" ?

### DISCUSSION THREAD

**Discussion by postylem** [manna-harbour/miryoku#268] [Q&A] [answered]
I'm a little confused about the Button layer.  In the reference manual [here](https://github.com/manna-harbour/miryoku/blob/9dc98b5d7fb2b88d375e7ce5ffbe1bfd5e2f5942/docs/reference/readme.org?plain=1#L72C287-L72C336), it says that the layer 

> can activated manually by holding a bottom row pinkie key (however `super` and `undo` will be unavailable).

I'm confused by this last comment this because I just built the layout (with QMK) with default configuration and it certainly seems like I can use undo for instance (e.g. hold left bottom pinkie <kbd>z</kbd> and tap right bottom pinkie <kbd>/</kbd>).

**Comment by KAGEYAM4** [comment, marked as answer]
maybe it meant on the same hand. because `undo` will be buried inside `button` layer key, and you wont be able to access `super` for which your require a `pinkie`.

**Reply by postylem** [reply]
Ah okay, that would make sense. Also there is another undo on the top row but also in the pinkie column. So they're still there, though not ergonomically accessible.

**Reply by manna-harbour** [reply]
Yes, that's right. It should instead say something like this.

>however `Super` and `Undo` will be *unreachable with one hand without using an alternative finger*
